### PR TITLE
java: fix crash with evmc_last_error_msg

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: bindings/java/c/evmc-vm.c [hunk 1]
```diff
@@ -24,7 +24,7 @@ JNIEXPORT jobject JNICALL Java_org_ethereum_evmc_EvmcVm_init(JNIEnv* jenv,
         {
             const char* error_msg = evmc_last_error_msg();
             jclass jclazz = (*jenv)->FindClass(jenv, "java/lang/AssertionError");
-            (*jenv)->ThrowNew(jenv, jclazz, error_msg);
+            (*jenv)->ThrowNew(jenv, jclazz, error_msg ? error_msg : "Loading EVMC VM failed");
```
Comment (axic) [inline bindings/java/c/evmc-vm.c]: `ThrowNew` expects a valid pointer and `evmc_last_error_msg` can return null.

**Comment by axic** [comment]
Have bunch of other refactorings too in the pipeline.

File: bindings/java/c/evmc-vm.c (hunk 1 above)
Comment (chfast) [reply bindings/java/c/evmc-vm.c]: Is there really a crash happening?

Comment (axic) [reply bindings/java/c/evmc-vm.c]: https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/functions.html

The documentation is not explicit about it, but says:
> message: the message used to construct the java.lang.Throwable object. The string is encoded in modified UTF-8.

**Pull request merged**